projects/wave-core-new/src/lib/operators/dialogs/expression-operator/expression-operator.component.ts

```diff
@@ -152,6 +153,43 @@ export class ExpressionOperatorComponent implements AfterViewInit, OnDestroy {
         // this.unitSubscription.unsubscribe();
     }
 
+    validateNoData(control: AbstractControl): ValidationErrors | null {
+        if (!isNaN(parseFloat(control.value)) || control.value.toLowerCase() === 'nan') {
```

Comment:
You can express no data as a number, but just not with JSON.
IMHO you should work with number here and as potential string for submitting it to our server.

Reply:
users have to be able to set NAN as the selected no data value